setup.py

```diff
@@ -2,25 +2,24 @@
 
 import sys
 
-from setuptools import find_packages
-from setuptools import setup
+from setuptools import find_packages, setup
 
 v = sys.version_info
 if v[:2] < (3, 6):
-    error = "ERROR: jupyterhub-kubespawner requires Python version 3.6 or above."
+    error = "ERROR: kubespawner requires Python version 3.6 or above."
     print(error, file=sys.stderr)
     sys.exit(1)
 
 setup(
-    name='jupyterhub-kubespawner',
-    version='2.0.1.dev',
+    name='kubespawner',
```

Comment:
Looks like the setup.py changes other than the version bumps are for your fork? Should be removed before it lands in the main repo.

Reply:
Thank you, yes.